doc/development/development_guide.rst

```diff
@@ -157,6 +198,21 @@ Test coverage can be checked by running
 The output of the above command will show the coverage percentage of each
 file, as well as the line numbers of any lines missing test coverage.
 
+To obtain coverage, the ``pytest-cov`` plugin is needed.
+
+The coverage of a specific file can also be checked by generating a report:
+
+.. code-block:: python
+
+    pytest tests/backend/test_states.py --cov --cov-report=term-missing
```

Comment:
it might be good to have
```suggestion
    pytest tests/backend/test_states.py --cov=strawberryfields/location/to/module --cov-report=term-missing
```

If not provided, the default will simply be `--cov=strawberryfields`, which means the coverage report will include a lot of 0% coverage for modules that were not tested.

Reply:
That's a good point! Added a note on that and extended the command